modules/administration-guide/partials/proc_deploying-registries-in-openshift.adoc

```diff
@@ -18,7 +18,7 @@ oc new-app -f openshift/che-plugin-registry.yml \
  -n "$\{NAMESPACE}" \
  -p IMAGE="$\{IMAGE_NAME}" \
  -p IMAGE_TAG="$\{IMAGE_TAG}" \
- -p PULL_POLICY="IfNotPresent"
+ -p PULL_POLICY="Always"
```

Comment:
For helm the pull policy of the registries is still IfNotPresent. Is there a reason for that difference?

Reply:
I believe it may cause issues, typically when I rebuild registry, I use the same tag. And IfNotPresent may lead to situation where my registry is not deployed but image is right. So, I believe always is safer

Reply:
I'll update helm chart to Always as well, if you don't mind

Reply:
Update to Always for helm chart as well. Thanks for catching it.